dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -242,6 +242,27 @@ export const ImageComponent = ({
 
 	const palette = decidePalette(format);
 
+	const master = element.media.allImages.find(
+		(image) => image.fields.isMaster,
+	)?.url;
+
+	// Legacy images do not have a master so we fallback to the largest available
+	const descendingByWidth = (a: Image, b: Image) => {
+		return parseInt(b.fields.width) - parseInt(a.fields.width);
+	};
+	const fallbackImage =
+		element.media.allImages.sort(descendingByWidth)[0].url;
+
+	const isSupported = (imageUrl: string) => {
+		const supportedImages = ['jpg', 'jpeg', 'png'];
+		const extension = imageUrl.split('.').pop();
+		return extension && supportedImages.includes(extension);
```

Comment:
Is there any risk that the image format is uppercase?

FYI, you can use `.slice(-1)` for an immutable version of `pop()`.

Reply:
Both fixed 👍